ui/components/stint_tracking/strategies/StrategyTab/helpers/_handle_sync_from_tracker_failed.py

```
@@ -0,0 +1,15 @@
+"""Handle background strategy sync failures."""
+
+from __future__ import annotations
+
+from core.errors import log
+
+
+def _handle_sync_from_tracker_failed(self, strategy_name: str) -> None:
```

Ruff (0.15.5)

[warning] Unused function argument: `self`

(ARG001)